app/AppKernel.php

```diff
@@ -43,7 +42,6 @@ public function registerBundles()
 
         if (in_array($this->getEnvironment(), array('dev', 'test'))) {
             $bundles[] = new Symfony\Bundle\WebProfilerBundle\WebProfilerBundle();
-            $bundles[] = new Sensio\Bundle\DistributionBundle\SensioDistributionBundle();
```

Comment:
Did you leave it in the Composerfile on purpose?

https://github.com/OpenConext/Stepup-SelfService/blob/996ed6af12794b2492eeb7e3a6db327c810f3f57/composer.json#L25